app/gui/src/model/suggestion_database/entry.rs

```diff
@@ -503,20 +542,20 @@ mod test {
     use super::*;
 
 
-    fn expect(
-        entry: &Entry,
-        current_module: Option<&module::QualifiedName>,
-        generate_this: bool,
-        expected_code: &str,
-        expected_imports: &[&module::QualifiedName],
-    ) {
-        let CodeToInsert { code, imports } = entry.code_to_insert(current_module, generate_this);
-        assert_eq!(code, expected_code);
-        assert_eq!(imports.iter().collect_vec().as_slice(), expected_imports);
-    }
-
     #[test]
     fn code_from_entry() {
+        fn expect(
+            entry: &Entry,
+            current_module: Option<&module::QualifiedName>,
+            generate_this: bool,
+            expected_code: &str,
+            expected_imports: &[&module::QualifiedName],
+        ) {
+            let CodeToInsert { code, imports } =
+                entry.code_to_insert(current_module, generate_this);
```

Comment:
use one line exprs. Instead of unwrapping the values you can access them later with dots.